config/eslint/.eslintrc-test

```diff
@@ -0,0 +1,12 @@
+---
+extends:
+  - "defaults/configurations/walmart/es5-node"
+
+env:
+  mocha: true
+
+rules:
```

Comment:
I grabbed them from the sibling `.eslint-react` config here: https://github.com/FormidableLabs/builder-docs-archetype/blob/d4bdd214882164dff34d4c660e93f3ea3f3293d5/config/eslint/.eslintrc-react

I just removed them, since those rules aren't needed for the specs. It's probably worth removing those rules from the `.eslint-react` as well and fixing downstream project errors.